docs/src/pages/index.js

```
@@ -167,14 +174,87 @@ const formSlides = [
   },
 ];
 
+/** The command the hero advertises, in pnpm's authoring vocabulary. */
+const HERO_COMMAND = 'add @allxsmith/bestax-bulma';
+
 function HomepageHeader() {
   const { siteConfig } = useDocusaurusContext();
```

ESLint

[error] 'siteConfig' is assigned a value but never used.

(no-unused-vars)